usmqe/web/tendrl/mainpage/clusters/cluster_list/pages.py

```diff
@@ -4,22 +4,53 @@
 
 
 import copy
+import pytest
 
 from webstr.patternfly.contentviews import pages as contentviews
 import webstr.patternfly.dropdown.pages as dropdown
 
+from usmqe.gluster import gluster
+from usmqe.ceph import ceph_cluster
 import usmqe.web.tendrl.mainpage.clusters.cluster_list.models as m_cluster_list
 from usmqe.web.tendrl.mainpage.clusters.import_cluster_wizard.pages\
     import ImportCluster, ImportClusterSummary
 from usmqe.web.tendrl.auxiliary.pages import ListMenu
-from usmqe.web.tendrl.mainpage.tasks.pages import TaskDetails
+
+
+def check_hosts(hosts_list, page_hosts_list):
+    """
+    check if all hosts in host_list are present on the page (in page_host_list)
+
+    Parameters:
+        hosts_list (list): list of dictionaries
+                          {'hostname': <hostname>, 'role': <role>, ...
+        page_hosts_list (list): list representing lines in the page hosts list
+    """
+    aux_list = copy.deepcopy(hosts_list)
+    for host_row in page_hosts_list:
+        found = False
+        for host in aux_list:
```

Comment:
I think that this iteration is wrong, see:
```
In [9]: l = [{1:1}, {2:2}, {3:3}, {4:4}]

In [10]: for i in l:
    print i
    l.remove(i)
   ....:     
{1: 1}
{3: 3}

In [11]: l
Out[11]: [{2: 2}, {4: 4}]
```
Please use ```hosts_list``` in iteration or use different algorithm.